src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2866,6 +2929,17 @@ sysroot_finalize_deployment (OstreeSysroot     *self,
         return FALSE;
     }
 
+  static const gchar * const SEMODULE_CMD[] = {
+    "semodule", "-N", "--rebuild-if-modules-changed"
+  };
+  gint exit_status;
+  if (!run_in_deployment(deployment_dfd, SEMODULE_CMD,
```

Comment:
Convention in this repo is

```suggestion
  if (!run_in_deployment (deployment_dfd, SEMODULE_CMD,
```

for function calls.